automatic/gimp/gimp.nuspec

```diff
@@ -16,9 +17,8 @@ GIMP has many capabilities. It can be used as a simple paint program, an expert
 
 GIMP is expandable and extensible. It is designed to be augmented with plug-ins and extensions to do just about anything. The advanced scripting interface allows everything from the simplest task to the most complex image manipulation procedures to be easily scripted.
     </description>
-    <summary>GIMP</summary>
-    <releaseNotes></releaseNotes>
-    <copyright>GPL</copyright>
+    <releaseNotes>http://www.gimp.org/release-notes/</releaseNotes>
```

Comment:
>  Any fully valid URI is automatically converted into a clickable link by markdownsharp

I did not know this!  Good to know!

>  try to just leave a static link for releaseNotes tag because there isn't a good way for now to automatically update the content in this section with chocopkgup

Yes, I do the same thing, but I write them as a fully declared markdown URL, i.e with [] and ()'s.  Will know that this is not required going forward.